src/util/handlebars/handlebar-helpers.ts

```diff
@@ -27,6 +27,9 @@ export const registerHandlebarsHelpers = () => {
 				: `${numberOfSyncedRepos} / ${totalNumberOfRepos}`
 	);
 
+	//check to display repo count as zero in case of org with zero repos
```

Comment:
nit comment duplicating code

Reply:
Either we should remove it or change the code to be expressive enough so it won't need comments describing what it does ;)